pkg/service/backup/service_backup_integration_test.go

```diff
@@ -430,7 +430,8 @@ func TestGetTargetIntegration(t *testing.T) {
 			if diff := cmp.Diff(golden, v,
 				cmpopts.SortSlices(func(a, b string) bool { return a < b }),
 				cmpopts.IgnoreUnexported(backup.Target{}),
-				cmpopts.IgnoreSliceElements(func(u backup.Unit) bool { return u.Keyspace == "system_replicated_keys" })); diff != "" {
+				cmpopts.IgnoreSliceElements(func(u backup.Unit) bool { return u.Keyspace == "system_replicated_keys" }),
+				cmpopts.IgnoreSliceElements(func(t string) bool { return t == "dicts" })); diff != "" {
```

Comment:
I can change that. In general I am not a big fan of the golden file tests - they seem more annoying than useful when working with different Scylla versions. Maybe it would be good to get rid of them in the future, or perhaps it would be better to separate tests for different Scylla versions.